From 76ff4ce956542827f5eff43f402aae0254e3ec3a Mon Sep 17 00:00:00 2001
From: Vyacheslav Bocharov <adeep@lexina.in>
Date: Fri, 10 Dec 2021 08:30:10 +0300
Subject: [PATCH 3/3] arm64: meson: dts: fix sdio in dts for JetHub D1

Fix the dts to match board's reference design:
- update vddio_boot regulator to 3.3v
- remove emmc hs200 support due to the lack of 1.8v regulator

Signed-off-by: Vyacheslav Bocharov <adeep@lexina.in>
---
 .../boot/dts/amlogic/meson-axg-jethome-jethub-j100.dts     | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-axg-jethome-jethub-j100.dts b/arch/arm64/boot/dts/amlogic/meson-axg-jethome-jethub-j100.dts
index 9951217ef997..2965346fc47e 100644
--- a/arch/arm64/boot/dts/amlogic/meson-axg-jethome-jethub-j100.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-axg-jethome-jethub-j100.dts
@@ -81,8 +81,8 @@ vddio_ao18: regulator-vddio_ao18 {
 	vddio_boot: regulator-vddio_boot {
 		compatible = "regulator-fixed";
 		regulator-name = "VDDIO_BOOT";
-		regulator-min-microvolt = <1800000>;
-		regulator-max-microvolt = <1800000>;
+		regulator-min-microvolt = <3300000>;
+		regulator-max-microvolt = <3300000>;
 		vin-supply = <&vddao_3v3>;
 		regulator-always-on;
 	};
@@ -276,8 +276,7 @@ &sd_emmc_c {
 	max-frequency = <200000000>;
 	non-removable;
 	disable-wp;
-	mmc-ddr-1_8v;
-	mmc-hs200-1_8v;
+	mmc-ddr-3_3v;
 
 	mmc-pwrseq = <&emmc_pwrseq>;
 
-- 
2.30.2

